### ISSUE TITLE
hangup with bluetooth button

### ISSUE THREAD

**Issue by 532910** [BelledonneCommunications/linphone-android#618]
Please add support for hanging up with bluetooth headset button.
While answering a call is not possible #574, hanging up can be implemented as https://github.com/jitsi/jitsi-meet does it.

**Comment by Viish** [comment]
Please add a link to the part of code where jitsi does that.

**Comment by 532910** [comment]
Sorry, I'm not an android developer, but looks like jitsi uses telecom so this issue should be merged with #574. 

Anyway I see in logcat:
```
D BluetoothManager: [Bluetooth] Event: +XEVENT : BUTTON, id = 2 (1)
I Telecom : BluetoothPhoneService: BT - hanging up call: BPSI.hC@ASo
```
